controller/src/beerocks/master/db/db.cpp

```diff
@@ -2298,6 +2298,11 @@ int db::get_channel_scan_dwell_time_msec(const sMacAddr &mac, bool single_scan)
 bool db::is_channel_scan_pool_supported(const sMacAddr &mac,
                                         const std::unordered_set<uint8_t> &channel_pool)
 {
+    if (channel_pool.size() == 1 && channel_pool.find(0) != channel_pool.end()) {
```

Comment:
you should use a define 

```suggestion
    if (channel_pool.size() == 1 && *channel_pool.begin() == SCANALLCHAN) {
```

Reply:
Changed